models/sipnet/R/read_restart.SIPNET.R

```diff
@@ -22,9 +22,14 @@ read_restart.SIPNET <- function(outdir, runid, stop.time, settings, var.names, p
   prior.sla <- params[[which(!names(params) %in% c("soil", "soil_SDA", "restart"))[1]]]$SLA
   
   forecast <- list()
-  # additional varnames, because we need these deterministic relationships
-  var.names <- c(var.names, "fine_root_carbon_content", "coarse_root_carbon_content")
-  
+  params$restart <-c() #state.vars not in var.names will be added here
+  #SIPNET inital states refer to models/sipnet/inst/template.param
+  state.vars <- c("SWE", "SoilMoistFrac", "AbvGrndWood", "TotSoilCarb", "LAI", 
```

Comment:
litter moisture seems to still be missing

Reply:
While litter moisture is written to the sipnet.out files, this variable appears to be absent from the .nc files. I've review model2netcdf.SIPNET and litter water is not included in the variables transfered from sipnet.out to .nc. Since the read.restart.SIPNET function reads in the .nc files, I did not include litter water. If litter water should be included then this will require an update to model2netcdf as well as read.restart.SIPNET.

Reply:
Was this a previous oversight? Should I update model2netcdf to include litter water?